cohortextractor/presto_utils.py

```diff
@@ -6,13 +6,21 @@
 import prestodb
 import requests
 from requests_pkcs12 import Pkcs12Adapter
+from retry import retry
 from tabulate import tabulate
 
 # TODO remove this when certificate verification reinstated
 import urllib3
 urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 
 
+LOG_SQL = bool(os.getenv("LOG_SQL"))
```

Comment:
Should prob either call this `LOG_PRESTO_SQL` or also add it to the TPP backend?

Reply:
This is all speculative, we can do it right later.